fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetaClient.java

```diff
@@ -395,6 +405,25 @@ private List<Partition> getPartitionsWithRetry(String dbName, String tableName,
         }
     }
 
+    public List<FieldSchema> getAvroFields(String dbName, String tableName, Map<String, String> parameters) {
```

Comment:
This method has nothing to do with accessing  hive metastore and could port to other classes.

Reply:
@stephen-shelby done